html/assets/compare/machinery.js

```diff
@@ -67,4 +67,11 @@ $(document).ready(function () {
     $scope.find(".show-common-elements").show();
     return false;
   });
+
+  $(".both-show-common-elements").click(function(){
```

Comment:
I would not use a "both-show-common-elements" but modify "show-common-elements" so it can handle both at the same.